dwyl/dwyl.xcodeproj/project.pbxproj

```diff
@@ -429,12 +429,14 @@
 					};
 					0D20381B1E6D82FD008CB3F4 = {
 						CreatedOnToolsVersion = 8.2.1;
+						DevelopmentTeam = 3T4BK2BXHV;
```

Comment:
Is there no way of making this a Constant to avoid having to update it in multiple places?
https://stackoverflow.com/questions/26252233/global-constants-file-in-swift